src/Microsoft.AspNet.OData.Shared/Formatter/Serialization/SelectExpandNode.cs

```diff
@@ -554,6 +680,9 @@ internal static void ValidatePathIsSupportedForExpand(ODataPath path)
         public static void GetStructuralProperties(IEdmStructuredType structuredType, HashSet<IEdmStructuralProperty> structuralProperties,
             HashSet<IEdmStructuralProperty> nestedStructuralProperties)
         {
+            // Be noted: this method is not used anymore. Keeping it unremoved is only for non-breaking changes.
+            // If any new requirement for such method, it's better to move to "EdmLibHelpers" class.
+
             if (structuredType == null)
```

Comment:
mark as obsolete?

Reply:
Yes. We can. However, I don't want to introduce breaking changes. Let's leave the change when we do V8?

---
In reply to: [355528099](https://github.com/OData/WebApi/pull/1941#discussion_r355528099) [](ancestors = 355528099)

Reply:
Marked it as obsolete. 

---
In reply to: [355619047](https://github.com/OData/WebApi/pull/1941#discussion_r355619047) [](ancestors = 355619047,355528099)